images/build/debian-iptables/distroless/get-binaries-from-package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
```

Comment:
My shell scripts are far from beauty than @BenTheElder and @thockin scripts, so feel free to suggest an all new script here folks!